content/accessibility/guidelines.mdx

```diff
@@ -105,11 +105,6 @@ For more detail on accessible links, read the W3C's [Understanding 2.4.4 - Link
 
 **Why?** Visible labels determine the purpose for inputs, and screen readers can read this information without extra code. If placeholder text contains important information and that information disappears when one starts to type, those with cognitive disabilities will have difficulty completing the form.
 
-#### Alt-text
-Use the HTML `alt` attribute when labeling an image. If the image has a descriptive caption already, you should give the image an attribute of `alt=""`. This will prevent the screenreader from telling the user about the image multiple times.
-
-**Why?** When screen readers come across images, graphs, and icons, by default the screen reader will speak the name of the file, which most of the time can be unhelpful (for example: `final-final-truly-final-V2.jpg`).
```

Comment:
Could we add this to the "Why" section of the new page? I like that it explains exactly what the experience is like for users if there is no `alt` attribute!

Reply:
I will include a paragraph about this, thanks :)